src/ca/team2706/vision/trackerboxreloaded/Main.java

```diff
@@ -306,34 +411,49 @@ public static void main(String[] args) {
         // Main video processing loop
         while (true) {
             if (useCamera) {
+            	//Read the frame from the camera, if it fails try again
                 if (!camera.read(frame)) {
                     System.err.println("Error: Failed to get a frame from the camera");
                     continue;
                 }
             } // else use the image from disk that we loaded above
-
-            if (use_GUI)
-                Imgproc.resize( frame, frame, visionParams.sz );
+            if(use_GUI)
+            //Resize the frame
+            Imgproc.resize( frame, frame, visionParams.sz );
```

Comment:
Hmm, I'm not sure if the `if` contains the `Imgproc.resize(..)`, or if it only contains the comment. Better add `{ }`'s to the `if` just to be safe.